src/test/suite/commands/getWorkspaceFromPathCommand.test.ts

```diff
@@ -0,0 +1,66 @@
+import * as chai from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import { GetWorkspaceFromPath, GetWorkspaceFromPathResult } from '../../../commands/getWorkspaceFromPath';
+import { CmShellMock } from './cmShellMock';
+
+chai.use(chaiAsPromised);
+
+describe('GetWorkspaceFromPathCommand', () => {
+  context('command successfully executed', () => {
+    it("parses a well-formed output", async () => {
+      const outputLines: string[] = ['wkname\t/path/to/wk\t95b0a429-7d9c-48af-8b5b-6f1ced257b20'];
```

Comment:
This testing output relies as well on chunks being always separated by a single `\t`.